File: Hahn.ApplicatonProcess.December2020.Web/Hahn.ApplicatonProcess.December2020.Data/Data/Applicant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace Hahn.ApplicatonProcess.December2020.Data.Data
{
    public class Applicant
    {
        [JsonIgnore]
        public int ID { get; set; }
        public string Name { get; set; }
        public string FamilyName { get; set; }
        public string Address { get; set; }
        public string CountryOfOrigin { get; set; }
        public string EMailAdress { get; set; }
        public int Age { get; set; }
        public bool Hired { get; set; }
    }
}

File: Hahn.ApplicationProcess.December2020.Client/src/Applicant/applicant.ts
export class Applicant {
  id: string;
  name: string;
  familyName: string;
  address: string;
  countryOfOrigin: string;
  eMailAdress: string;
  age:number;
  hired:boolean

  constructor(data) {
      Object.assign(this, data);
  }

  /**getAddress() {
      return `${this.address} ${this.city}, ${this.state} ${this.postalCode}`;
  }
  **/

}
